src/routes/Network.js

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
 import PendingTradeRequests from '../components/PendingTradeRequests'
+import StatusTradeRequests from '../components/StatusTradeRequests'
+import ToggleSwitch from '@trendmicro/react-toggle-switch';
 
-export default (props) => (
-  <PendingTradeRequests uid={props.data.user.id}></PendingTradeRequests>
-)
+const Network= (props) => {
+  const { user } = props.data;
+
+  return (
+    props.data.loading
+      ? <div>Loading</div>
+      : <div>
+          <PendingTradeRequests uid={user.id}></PendingTradeRequests>
+          <br/>
```

Comment:
Let's remove this line-break. If we need to add some styling, let's do it with CSS.